submissions/Cashass/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,190 @@
+                        />Контролер</a
+                      >
+                    </li>
+                    <li class="popup__list">
+                      <a href="#" class="popup__link">
+                        <img
+                          class="popup-img"
+                          src="icon/Мапа.png"
+                          alt
+                        />Мапа</a
+                      >
+                    </li>
+                    <li class="popup__list">
+                      <a href="#" class="popup__link">
+                        <img
+                          class="popup-img"
+                          src="icon/Новини.png"
+                          alt
+                        />Новини</a
+                      >
+                    </li>
+                    <li class="popup__list">
+                      <a href="#" class="popup__link">
+                        <img
+                          class="popup-img"
+                          src="icon/Нотатки.png"
+                          alt
+                        />Нотатки</a
+                      >
+                    </li>
+                    <li class="popup__list">
+                      <a href="#" class="popup__link">
+                        <img
+                          class="popup-img"
+                          src="icon/Поділитися.png"
+                          alt
+                        />Поділитися</a
+                      >
+                    </li>
+                  </ul>
+                  <input
+                    type="checkbox"
+                    class="sub__check-input"
+                    id="sub__input"
+                  />
+                  <label for="sub__input" class="sub__check-label">
+                    Більше
+                  </label>
+                  <div class="sub__lists-popup">
+                    <ul class="sub__lists">
+                      <li class="popup__list">
+                        <a href="#" class="popup__link">
+                          <img
+                            class="sub__popup-img"
+                            src="icon/Поради.png"
+                            alt
+                          />Поради</a
+                        >
+                      </li>
+                      <li class="popup__list">
+                        <a href="#" class="popup__link">
+                          <img
+                            class="sub__popup-img"
+                            src="icon/Пошта.png"
+                            alt
+                          />Пошта</a
+                        >
+                      </li>
+                      <li class="popup__list">
+                        <a href="#" class="popup__link">
+                          <img
+                            class="sub__popup-img"
+                            src="icon/Месенджер.png"
+                            alt
+                          />Месенджер</a
+                        >
```

Comment:
alt must be equal to the empty string.
alt=''

Reply:
Corrected. Changed file names to English. Add alt="". May I clarify? Can I leave a description of the "menu" and "View" icons? They carry meaning.

Reply:
> Can I leave a description of the "menu" and "View" icons? They carry meaning.
Yes, you can. 
If there is no text label on the menu button, you need to add alt-text. You can also describe its functionality in the code itself, like aria-label